onnxscript/type_annotation.py

```diff
@@ -43,6 +43,23 @@
 
 _LIST_CONSTRUCTORS = frozenset([list, typing.List, typing.Sequence, collections.abc.Sequence])
 
+# Map from ONNX AttributeProto type to its representation (in ONNX Script).
+_ATTRTYPE_TO_REPR = {
+    onnx.AttributeProto.FLOAT: "float",
+    onnx.AttributeProto.INT: "int",
+    onnx.AttributeProto.STRING: "str",
+    onnx.AttributeProto.FLOATS: "Sequence[float]",
+    onnx.AttributeProto.INTS: "Sequence[int]",
+    onnx.AttributeProto.STRINGS: "Sequence[str]",
+}
+
+
+def onnx_attr_type_to_onnxscript_repr(attr_type: onnx.AttributeProto.AttributeType) -> str:
+    if attr_type not in _ATTRTYPE_TO_REPR:
+        raise ValueError("Unsupported attribute type {attr_type}")
```

Comment:
Include the supported in the error message?

Reply:
Done